e2e-app/src/app/datepicker/multiple/datepicker-multiple.e2e-spec.ts

```diff
@@ -0,0 +1,20 @@
+import {Key} from 'protractor';
+import {sendKey, openUrl} from '../../tools.po';
+import {DatepickerMultiplePage} from './datepicker-multiple.po';
+
+describe('Datepicker', () => {
+  let page: DatepickerMultiplePage;
+
+  beforeAll(() => page = new DatepickerMultiplePage());
+
+  beforeEach(async() => await openUrl('datepicker/multiple'));
+
+  it('should lose focus on tapping on a different datepicker instance', async() => {
+    await page.getDayElement(new Date(2016, 7, 1), 0).click();
+    await sendKey(Key.ARROW_DOWN);
+    await page.expectActive(new Date(2016, 7, 8), 0);
+    await page.getDayElement(new Date(2016, 7, 1), 1).click();
+    await sendKey(Key.ARROW_DOWN);
+    await page.expectActive(new Date(2016, 7, 8), 1);
```

Comment:
could you please check here that the day is actually focused? there is an `expectFocused()` utility for this
It still can be active, but not focused